lib/url.c

```diff
@@ -851,7 +851,6 @@ CURLcode Curl_disconnect(struct Curl_easy *data,
    */
   if(CONN_INUSE(conn) && !dead_connection) {
     DEBUGF(infof(data, "Curl_disconnect when inuse: %zu", CONN_INUSE(conn)));
-    return CURLE_OK;
   }
```

Comment:
"Applying suggestions on deleted lines is not supported", ok github.

```suggestion
    return;
  }
```

Reply:
oops, good catch, yes indeed!